crates/bevy_window/Cargo.toml

```diff
@@ -18,6 +18,8 @@ bevy_app = { path = "../bevy_app", version = "0.5.0" }
 bevy_ecs = { path = "../bevy_ecs", version = "0.5.0" }
 bevy_math = { path = "../bevy_math", version = "0.5.0" }
 bevy_utils = { path = "../bevy_utils", version = "0.5.0" }
+bevy_asset = { path = "../bevy_asset", version = "0.5.0" }
+# bevy_render = { path = "../bevy_render", version = "0.5.0" }
```

Comment:
Yes, it should be. 
But I am running into dependency problems (cyclic), because bevy_render depends on bevy_window for its camera specific stuff. 
From an outside perspective of a noob, there seem to be 2 naive solutions:
1. Extract the camera code into a separate crate, so that bevy_render does not need to depend on bevy_window 
2. Extract the texture code into a separate crate, so that bevy_window does not need to depend on bevy_render

I tried my hand at the second solution, but there some conversion functions in texture that I could not extract. (Maybe they could be implemented via a trait in bevy_render?). But it did not look very nice anyways. 
Solution 1 I initially disregarded, but looking at it again, it might be a possibility. After all the theoretical "bevy_camera" could depend on bevy_render and bevy_window without a problem (hopefully :)).

In general I am not sure if this is worth the trouble though. I will go into a bit more detail in a separate comment.

Reply:
I'm not a rendering expect, but I don't think removing `Texture` from `bevy_render` makes very much sense, since they go hand in hand.

However, if we are able to extract the camera code into it's own crate, I think that would be the best path forward.

Reply:
bevy_camera is possible I think, but there are other parts in rendering that depend on window, that I initially overlooked. 
I will try to implement the proposal from cart.